contracts/GolemNetworkTokenWrapped.sol

```diff
@@ -188,7 +191,7 @@ contract GolemNetworkTokenWrapped is Token {
     // This function allows batch payments using sent value and
     // sender's balance.
     // Cost: 21000 + (5000 + ~2000) * n
```

Comment:
This cost computation is wrong since the code has changed. It needs to be done again.

Reply:
Done